### ISSUE TITLE
TCA enable fields not configurable

### ISSUE THREAD

**Issue by garbast** [in2code-de/luxletter#202]
In TYPO3 12  `'rootLevel' => -1,` disables the possibilily to configure enable fields in backend user groups, leaving users unable to edit records like sender configuration.

Please remove said rootLevel config.

**Comment by einpraegsam** [comment]
Sounds like the issue on forge: https://forge.typo3.org/issues/101633
As far as I remember, the settings is useful as it is.

**Comment by garbast** [comment]
Yes, that is the problem. But until this is fixed in the core, it would be good to have it changes in the extension table TCA, to allow the usage of this tables.

**Comment by einpraegsam** [comment]
There will no change in the next days to this. Before we do a workarround for a TYPO3 bug, we discuss this in the team. For now pls overwrite the TCA in your sitepackage.

**Comment by garbast** [comment]
Thanks for taking care.

I understand, that the better solution would be to have the core bug fixed, but for now it's working again from a backend editors perspective.

**Comment by einpraegsam** [comment]
yes I agree with you :)
